reference/store_information.v2.yml

```diff
@@ -98,9 +100,234 @@ paths:
+        '422':
+          description: |
+            Response object for metafields deletion with partial success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetaFieldCollectionResponsePartialSuccess_DELETE'
+  '/store/metafields/{metafieldId}':
+    get:
+      summary: Get a Store Metafield
+      tags:
+        - Metafields
+      description: Gets a store `metafield`.
+      operationId: getStoreMetafield
+      responses:
+        '200':
+          description: |
+            A `Metafield` object.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetafieldResponse'
+        '409':
+          description: |
+            The `Metafield` conflicts with another `Metafield`. This can result from duplicate unique key combinations of the appʼs client id, namespace, key, resource_type, and resource_id.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '422':
+          description: |
+            The `Metafield` is not valid. This is the result of missing required fields or of invalid data. See the response for more details.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal Server Error
+      parameters:
+        - $ref: '#/components/parameters/PageParam'
+        - $ref: '#/components/parameters/LimitParam'
+        - $ref: '#/components/parameters/MetafieldKeyParam'
+        - $ref: '#/components/parameters/MetafieldNamespaceParam'
+        - $ref: '#/components/parameters/DirectionParam'
+    put:
+      summary: Update a Store Metafield
+      tags:
+        - Metafields
+      description: |
+        Update a store `Metafield`.
+      operationId: updateStoreMetafield
+      parameters:
+        - $ref: '#/components/parameters/ContentType'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MetafieldBase_Post'
+        description: |
+          A `Metafield` object.
+        required: true
+      responses:
+        '200':
+          description: |
+            A metafield and metadata.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetaFieldCollectionResponse'
+        '404':
+          description: |
+            The resource was not found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotFound'
+              example:
+                value:
+                  status: 404
+                  title: There was no metafield found with ID 1010
+                  type: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
+    delete:
```

Comment:
- `Accept` header is not required
- It is possible to have `404` status

![image](https://github.com/bigcommerce/docs/assets/33040515/9c6d2a11-5a06-45a1-a447-771f23fa3b6e)